app/presenters/curation_concerns/presenter_factory.rb

```diff
@@ -30,7 +30,16 @@ def build
 
       # @return [Array<SolrDocument>] a list of solr documents in no particular order
       def load_docs
-        ActiveFedora::SolrService.query("{!terms f=id}#{ids.join(',')}", rows: 1000).map { |res| SolrDocument.new(res) }
+        query("{!terms f=id}#{ids.join(',')}", rows: 1000)
+          .map { |res| SolrDocument.new(res) }
+      end
+
+      # Query solr using POST so that the query doesn't get too large for a URI
+      def query(query, args = {})
+        args.merge!(q: query, qt: 'standard', rows: 1000)
```

Comment:
Visually I kinda prefer it on line 39. Line 33 is already a bit eye-popping. But meh, either is fine.